emperor/support_files/js/opacity-view-controller.js

```diff
@@ -72,10 +83,21 @@ define([
 
     // webgl acts up with transparent objects, so we only set them to be
     // explicitly transparent if the opacity is not at full
-    _.each(dv.markers, function(element) {
-      element.material.transparent = transparent;
-      element.material.opacity = value;
-    });
+    if (dv.decomp.isScatterType()) {
+      _.each(dv.markers, function(element) {
```

Comment:
similar comment here w.r.t. the anonymous function

Reply:
Fixed with the approach you described above.